### PR TITLE
Updated Italian translation

### PR DESCRIPTION
**Pull request by Kuddus73**
It updates the Italian translation.

### CODE REVIEW

File: lang/Italiano.json
Comment (Windows200000) [inline lang/Italiano.json:1]: Thanks! Could you please elaborate on the changes in the second half? I started tracking this in #2

Comment (Kuddus73) [reply lang/Italiano.json:1]: I believe the original was a straight translation from English to Italian, probably using a translator.
The changes in the second half make it more lexically and natively correct, that's all.

Comment (Kuddus73) [reply lang/Italiano.json:1]: I believe the original was a straight translation from English to Italian, probably using a translator.
The changes in the second half make it more lexically and natively correct and less "robotic", that's all.

Comment (Windows200000) [reply lang/Italiano.json:1]: Thanks for your contribution! Gonna put this to in-dev and wait until more accumulates, before building new binaries.

**Pull request merged**